src/iOS/toga_iOS/libs/core_graphics.py

```diff
@@ -1,6 +1,114 @@
 ##########################################################################
 # System/Library/Frameworks/CoreGraphics.framework
 ##########################################################################
+from ctypes import *
```

Comment:
At column 1: (F403) 'from ctypes import *' used; unable to detect undefined names